File: java/42 MaxMinArray.java
package code;
import java.util.*;
public class Test {
	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		int arr[]= {1,2,3,4,5,6,7,8,9};
		int min=arr[0];
		int max=arr[0];
		for(int i=0;i<arr.length;i++) {
			if(min>arr[i]) {
				min=arr[i];
			}
			if(max<arr[i]) {
				max=arr[i];
			}
		}
		System.out.println("MAX is "+max+" and MIN is "+min);
	}
}

File: python/42 MaxMinArray.py
import numpy as np
arr=np.array([1,2,3,4,5,6,7,8,9])
max=arr[0]
min=arr[0]
for val in arr:
    if val>max:
        max=val
    if val<min:
        min=val
print(f"MAX is {max} and MIN is {min}")